assignments/objects.js

```diff
@@ -49,16 +111,38 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  "name":"Susan",
+  "age":70,
+  "speak":function(){
+    console.log(`I am ${this.name}`);
+  },
+  "child":{
+    "name":"George",
+    "age":50,
+    "speak":function(){
+      console.log(`I am ${this.name}`);
+    },
+    "grandChild":{
+      "name":"Sam",
+      "age":30,
+      "speak":function(){
+        console.log(`I am ${this.name}`);
+      }
+    }
+  }
+}
 
 // Log the parent object's name
-
+console.log(parent);
 // Log the child's age
-
+console.log(parent.child.name);
 // Log the name and age of the grandchild
-
+console.log(parent.child.grandChild.name);
+console.log(parent.child.grandChild.age);
 // Have the parent speak
-
+console.log(parent.speak());
 // Have the child speak
-
+console.log(parent.child.speak());
```

Comment:
For your `speak` methods, no need to use `console.log` because the methods will return an `undefined` value - just invoke them directly instead.